util/src/harvey/cmd/elf2c/elf2c.go

```diff
@@ -139,7 +139,7 @@ func main() {
 		gencode(w, file, "code", mem, codestart, codeend)
 		gencode(w, file, "data", mem, datastart, dataend)
 	}
-	if err := ioutil.WriteFile(a[0], w.Bytes(), 0444); err != nil {
+	if err := ioutil.WriteFile(a[0], w.Bytes(), 0666); err != nil {
```

Comment:
Is this related?

Reply:
aaaah no, it came from the annoying rm init.h I have to make every time.